yellowbrick/features/jointplot.py

```diff
@@ -129,13 +129,14 @@ def __init__(self, ax=None, feature=None, target=None,
                  xy_plot='hist', xy_args=None,
                  size=6, ratio=5, space=.2, **kwargs):
 
-        #check matplotlib version - needs to be version 2.0.0
-        if mpl.__version__ == "2.0.0":
+        # Check matplotlib version - needs to be version 2.0.0 or greater.
+        mpl_vers_maj = int(mpl.__version__.split(".")[0])
```

Comment:
It seems like it might be a good idea to put a helper function in `version.py`, e.g. something like `parse_version` so that we can reuse this in multiple places and also add exception handling (`int()` can throw a `TypeError`). 

Not for this PR, but perhaps you could also write an issue or a TODO for a `check_version` function that raises a warning or exception based on a version of a specific dependency? I think it might be something that comes up more, and other libraries have deprecation warning decorators and the like.